tests/e2e/Services/Databases/DatabasesCustomServerTest.php

```diff
@@ -593,6 +593,200 @@ public function testUpdateCollection(array $data)
+        ]);
+
+        $this->assertEquals(201, $actors['headers']['status-code']);
+        $this->assertEquals($actors['body']['name'], 'Encrypted Actors Data');
+
+        /**
+         * Test for creating encrypted attributes
+         */
+
+        $attributesPath = '/databases/' . $databaseId . '/collections/' . $actors['body']['$id'] . '/attributes';
+
+        $firstName = $this->client->call(Client::METHOD_POST, $attributesPath . '/string', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), [
+            'key' => 'firstName',
+            'size' => 256,
+            'required' => true,
+        ]);
+
+        $lastName = $this->client->call(Client::METHOD_POST, $attributesPath . '/string', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), [
+            'key' => 'lastName',
+            'size' => 256,
+            'required' => true,
+            'encrypt' => true,
+        ]);
+
+        $age = $this->client->call(Client::METHOD_POST, $attributesPath . '/integer', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), [
+            'key' => 'age',
+            'min' => 0,
+            'max' => 120,
+            'required' => false,
+        ]);
+
+        $alias = $this->client->call(Client::METHOD_POST, $attributesPath . '/string', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), [
+            'key' => 'alias',
+            'size' => 256,
+            'required' => false,
+        ]);
+
+     /**
+      * Check Status of every Attribute
+      */
+        $this->assertEquals(202, $firstName['headers']['status-code']);
+        $this->assertEquals('firstName', $firstName['body']['key']);
+        $this->assertEquals('string', $firstName['body']['type']);
+
+        $this->assertEquals(202, $lastName['headers']['status-code']);
+        $this->assertEquals('lastName', $lastName['body']['key']);
+        $this->assertEquals('string', $lastName['body']['type']);
+
+        $this->assertEquals(202, $alias['headers']['status-code']);
+        $this->assertEquals('alias', $alias['body']['key']);
+        $this->assertEquals('string', $alias['body']['type']);
+
+        $this->assertEquals(202, $age['headers']['status-code']);
+        $this->assertEquals('age', $age['body']['key']);
+        $this->assertEquals('integer', $age['body']['type']);
+
+        // Wait for database worker to finish creating attributes
+        sleep(5);
```

Comment:
We can probably reduce this to 2 seconds, always good to reduce test time where we can